tools/mpremote/mpremote/commands.py

```diff
@@ -338,7 +359,7 @@ def do_filesystem(state, args):
                 if command == "cp":
                     print("{} {} {}".format(command, path, cp_dest))
                 else:
-                    print("{} :{}".format(command, path))
+                    print("{} {}:{}".format(command, "-r " if args.recursive else "", path))
```

Comment:
The `cp` command doesn't echo the `-r` above when recursive is used, so I don't think `rm` needs to either.  It'll be obvious from the verbose output that it's recursive.

Reply:
adjusted , including the .exp files